docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,586 @@
+# Creating a VPN connection between regions
+
+Thanks to the Openstack Neutron VPN as a Service (VPNaaS) feature, you 
+can bridge two different regions via a site-to-site IPSec VPN 
+connection. This is made possible without setting up and configuring a 
+virtual machine in any one of the regions. On the contrary, you can 
+quickly establish such a connection using the {{gui}} or the OpenStack 
+CLI. Let us demonstrate the process following both approaches.
+
+## Prerequisites
+
+Whether you choose to work from the {{gui}} or with the OpenStack CLI, 
+you need to [have an account](/howto/getting-started/create-account) in 
+{{brand}} Cloud. If you prefer to work with the [OpenStack 
+CLI](/howto/getting-started/enable-openstack-cli), then in addition to 
+the Python `openstackclient` module, you need to install the 
+corresponding plugin module for Neutron. Use either the package manager 
+of your operating system or `pip`:
+
+=== "Debian/Ubuntu"
+	```bash
+	apt install python3-neutron-vpnaas
+	```
+=== "Mac OS X with Homebrew"
+    This Python module is unavailable via `brew`, but you can 
+    install it via `pip`.
+=== "Python Package"
+	```bash
+	pip install neutron-vpnaas
+	```
```

Comment:
Errr, no. `neutron-vpnaas` is the *server* component implementing the VPNaaS API; it is not needed for client-side API VPNaaS management.

The VPNaaS client functionality is in `python-neutronclient`, which is a hard dependency of `python-openstackclient`. So as long as the OpenStack client is installed and the server supports Neutron VPNaaS, no additional package installation on the client is required.

Reply:
I mixed things up a bit, and now I see that I should not have installed `python3-neutron-vpnaas` (Debian/Ubuntu) or `neutron-vpnaas` (Mac OS). But since I clearly remember the command `vpn` being unavailable for `openstack`, I took a closer look and realized that `python-neutronclient` is *not* a hard dependency of `python-openstackclient`. Actually, in Debian or Ubuntu, in addition to `python3-openstackclient`, you have to install `python3-neutronclient`. And in `Mac OS`, in addition to `python-openstackclient`, you have to install `python-neutronclient`. I will update the HOWTO accordingly.

Reply:
Oh, interesting! Maybe that dependency changed (semi-)recently and I missed it. At any rate, the Neutron _client_ library it is. :)